README.md

```diff
@@ -1,6 +1,6 @@
 # ipaddr: IP (and MAC) address manipulation
 
-A library for manipulation of IP (and MAC) address representations.
+A library for manipulation of IP and MAC address representations.
```

Comment:
in line 1, we should remove the parens as well imho

Reply:
thanks updated the README more generally in 8e2fc40adc3fc944fb658fa1488e5d4b42cdba96